components/script/script_thread.rs

```diff
@@ -3828,6 +3832,27 @@ impl ScriptThread {
         context.process_response_eof(Ok(ResourceFetchTiming::new(ResourceTimingType::None)));
     }
 
+    // refer definition of pre_page_load method in "script_thread.rs"; code taken from "start_page_load_about_blank"
```

Comment:
Let's add this comment instead:
`/// Synchronously parse a srcdoc document from a giving HTML string.`